MNI_ecosystem.html

```diff
@@ -459,7 +364,7 @@ <h2>Visualization with Atelier 3D</h2>
 
 	</section>   
 
-	<!--PIVT-->
+	<!--PIVT Column 6-->
```

Comment:
- [ ] the PIVT image below is broken. 

this slide should say : 
PIVT : Platform-independent Visualization Tool
custom Javascript libraries for 
moving Neuroimaging to the web.
Easily integrated into LORIS, CBRAIN, and other platforms. 

(new slide)   
Visualizing Series data for Electrophysiology and EEG 
Javascript libraries in modular React components

[new screenshot - see below] 
(2-line caption:) The MNI Open iEEG Atlas in LORIS
https://mni-open-ieegatlas.research.mcgill.ca/

![Screen Shot 2019-10-28 at 5 13 55 PM](https://user-images.githubusercontent.com/827619/67718594-75567b00-f9a6-11e9-885b-e3993791fde6.png)

Reply:
this is in answer to your PIVT question https://github.com/samirdas/samirdas.github.io/pull/9#commitcomment-35629719

Reply:
done, thanks for this info